ci-operator/config/openshift-eng/rebasebot/openshift-eng-rebasebot-main.yaml

```diff
@@ -554,6 +554,37 @@ tests:
         requests:
           cpu: 100m
           memory: 200Mi
+- as: jaeger-operator
+  cron: 0 23 * * 1-5
+  steps:
+    test:
+    - as: jaeger-operator
+      commands: |
+        rebasebot --source https://github.com/jaegertracing/jaeger-operator:main \
+                  --dest openshift/jaegertracing-jaeger-operator:main \
+                  --rebase IshwarKanse/jaeger-operator:rebase-bot-main \
```

Comment:
Is there a shared org that this could push to rather than an individual's fork?

Reply:
For the time being we need to use a personal fork as an intermediate repo. There is one org that my team uses which has midstream repos, I'll need to check if we will be continue to use it or switch to the forks in openshift org for midstreams.

Reply:
Were you able to check this @IshwarKanse ?

Reply:
Ok got it thanks. Please do remember to update this as soon as you have settled on a team repo.
Thanks